src/guide/deployment/node-installation.md

```
@@ -24,21 +24,19 @@ This section will provide instructions for deploying the FLAME node software on
 
 ### Software
 
-::: tip
-A quick start guide to installing microk8s and Helm can be found [here](./microk8s-quickstart).
+::: tip A quick start guide to installing microk8s and Helm can be found [here](./microk8s-quickstart).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)